CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ option(FOUNDATION_PATH_TO_LIBDISPATCH_SOURCE "Path to libdispatch source" "")
 option(FOUNDATION_PATH_TO_LIBDISPATCH_BUILD "Path to libdispatch build" "")
 option(FOUNDATION_PATH_TO_XCTEST_BUILD "Path to XCTest build" "")
 
-find_package(CURL REQUIRED)
+find_package(CURL COMPONENTS libz REQUIRED)
```

Comment:
Don't think that this works on Windows :-(

Reply:
That's what I wanted to double check. Since there's a test that needs curl to support libz, we have to figure out a way to make it work.

I was actually more interested in the bit that detects if we are linking statically against cURL, and bringing libz along for the linkage of `FoundationNetworking`.